### PR TITLE
US036 API To AWS

### PR DESCRIPTION
**Pull request by Cormac-F**
- Created the Dockerfile and docker.yml to build the application within AWS

Screenshot 2023-12-13 at 10 58 00 am

### CODE REVIEW

**Review by lullabelle — CHANGES_REQUESTED** [review]
Hi
Need a review from team member before we approve
Also still not working!
See application logs below:
[application-logs2023-12-13T10_25_48.373Z-2023-12-14T10_25_48.373Z.json](https://github.com/Cormac-F/infinite-harmony-api/files/13672063/application-logs2023-12-13T10_25_48.373Z-2023-12-14T10_25_48.373Z.json)

**Review by CharlesM988 — APPROVED** [review]
LGTM!

**Review by lullabelle — APPROVED** [review]

**Pull request merged**
